samples/sample_10_windows/pipeline.json

```diff
@@ -0,0 +1,170 @@
+{
+  "pipeline": [
+    {
+      "name": "pipeline_clone_0",
+      "alias": "git",
+      "steps": [
+        {
+          "name": "pipeline_clone_0",
+          "alias": "git",
+          "image": "plugins/git:windows",
+          "working_dir": "c:/gopath/src/github.com/drone/envsubst",
+          "environment": {
+            "CI": "drone",
+            "CI_BUILD_CREATED": "1486119586",
+            "CI_BUILD_EVENT": "push",
+            "CI_BUILD_NUMBER": "6",
+            "CI_BUILD_STARTED": "1486119585",
+            "CI_COMMIT_AUTHOR": "bradrydzewski",
+            "CI_COMMIT_AUTHOR_NAME": "bradrydzewski",
+            "CI_COMMIT_BRANCH": "master",
+            "CI_COMMIT_MESSAGE": "added a few more test cases for escaping behavior",
+            "CI_COMMIT_REF": "refs/heads/master",
+            "CI_COMMIT_SHA": "d0876d3176965f9552a611cbd56e24a9264355e6",
+            "CI_REMOTE_URL": "https://github.com/drone/envsubst.git",
+            "CI_REPO": "drone/envsubst",
+            "CI_REPO_LINK": "https://github.com/drone/envsubst",
+            "CI_REPO_NAME": "drone/envsubst",
+            "CI_REPO_REMOTE": "https://github.com/drone/envsubst.git",
+            "CI_SYSTEM": "pipec",
+            "CI_SYSTEM_ARCH": "windows/amd64",
+            "CI_SYSTEM_LINK": "https://github.com/cncd/pipec",
+            "CI_SYSTEM_NAME": "pipec",
+            "CI_WORKSPACE": "c:/gopath/src/github.com/drone/envsubst",
+            "DRONE": "true",
+            "DRONE_ARCH": "linux/amd64",
```

Comment:
The `DRONE_ARCH` is being generated wrong.

Reply:
I did not notice it.
Is there variable likes HOST_OS_VERSION ? if there are no variable then how I can add this?
I think that will be a problem.
https://docs.microsoft.com/en-us/virtualization/windowscontainers/deploy-containers/version-compatibility

So I want use like this for image name : windows/amd64-${HOST_OS_VERSION} for windows/amd64-14393

Reply:
I think the yaml needs `platform: windows/arm64` as a top level attribute